app/controllers/events_controller.rb

```diff
@@ -14,10 +14,12 @@ def show
   # GET /events/new
   def new
     @event = Event.new
+    @event_types = EventType.all.pluck(:name, :id)
```

Comment:
In dynamically typed languages we expect variables to contain what their names suggests.
In this instance we expect an array of `EventType` records, but we get an array of arrays which contains only two values of the model.
Theory aside, if I understand correctly, this is line is only needed to event form. So we could move this logic to the form partial.
In most cases we try to resist putting logic into views but partials can be exceptions. Partials meant to be reusable so the logic need for them to work should be provided.
I made the suggestion in the corresponding file.
```suggestion
```